R/virtualenv.R

```diff
@@ -180,6 +182,21 @@ virtualenv_exists <- function(envname) {
 
 virtualenv_path <- function(envname) {
 
+  # if envname is NULL, check for an active renv environment
+  # and use its virtual environment if available
+  if (is.null(envname)) {
+    renv <- Sys.getenv("RENV_PROJECT", unset = NA)
+    if (!is.na(renv)) {
+      path <- file.path(renv, "renv/r-reticulate")
```

Comment:
We will probably want to make the `py_install()` envname also `NULL`, and then automatically create this virtualenv during py_install. There should also be an option to auto-create the environment when the renv is created.